### PR TITLE
July 2024 cleanup: folders 'v' to 'z', 'tokenscripts'

### PR DESCRIPTION
**Pull request by dracontes**

### CODE REVIEW

**Review by tool4ever — APPROVED** [review]

File: forge-gui/res/cardsfolder/v/volcano_hellion.txt [hunk 1]
```diff
@@ -5,7 +5,7 @@ PT:6/5
 Text:CARDNAME has echo X, where X is your life total.
 K:Echo:X
 T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigDmg | TriggerDescription$ When CARDNAME enters the battlefield, it deals an amount of damage of your choice to you and target creature. The damage can't be prevented.
-SVar:TrigDmg:DB$ ChooseNumber | ListTitle$ an amount of damage to you and target creature | ChooseAnyNumber$ True | SubAbility$ DBDmg | AILogic$ Min
+SVar:TrigDmg:DB$ ChooseNumber | ListTitle$ Choose amount of damage to be dealt | ChooseAnyNumber$ True | SubAbility$ DBDmg | AILogic$ Min
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/v/volcano_hellion.txt:8]: these changes need adjustment:
image

Comment (dracontes) [reply forge-gui/res/cardsfolder/v/volcano_hellion.txt]: Oops. Hopefully fixed to satisfaction. 
I took my lead from the pattern established `ChoiceTitle$`. I do hope either that one or `RevealTitle$` aren't amiss.

Comment (tool4ever) [reply forge-gui/res/cardsfolder/v/volcano_hellion.txt]: thanks
I'd have to check ingame but I'm gonna slowly start merging the other PR
feel free to update them before or in follow-up

**Review by tool4ever — APPROVED** [review]

File: forge-gui/res/cardsfolder/v/volcano_hellion.txt (hunk 1 above)
Comment (dracontes) [reply forge-gui/res/cardsfolder/v/volcano_hellion.txt]: Had a look to how things were on my side, and no surprises for those as far as I could tell.

**Pull request merged**